src/MetaMorpho.sol

```diff
@@ -275,6 +275,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         Id id = marketParams.id();
         if (marketParams.loanToken != asset()) revert ErrorsLib.InconsistentAsset(id);
         if (MORPHO.lastUpdate(id) == 0) revert ErrorsLib.MarketNotCreated();
+        if (config[id].removableAt != 0) revert ErrorsLib.RemovalSubmitted();
```

Comment:
I believe being consistent with the current code would require this check to be moved in the `else` branch below

Like the curator role can always instantly set a new cap, even if the market is submitted for removal

The cap being set at 0 upon removal submission makes this unnecessary, because the curator role can only increase the cap from here

So the only benefit would be 1 less SLOAD involved when curator role wants to instantly lower a cap on a market

Reply:
Yes I agree